openupgrade_scripts/scripts/hr_recruitment/18.0.1.1/post-migration.py

```diff
@@ -0,0 +1,44 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Create candidate records from applicant
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO hr_candidate (
+            partner_name, email_from, email_cc, partner_phone,
+            linkedin_profile, type_id, priority,
+            availability, create_date, write_date
+        )
+        SELECT partner_name, email_from, email_cc, partner_phone,
+               linkedin_profile, type_id, priority,
+               availability, NOW(), NOW()
+        FROM hr_applicant
+        WHERE candidate_id IS NULL;
```

Comment:
You can take these fields from the original record.
```suggestion
        INSERT INTO hr_candidate (
            partner_name, email_from, email_cc, partner_phone,
            linkedin_profile, type_id, priority,
            availability, create_date, write_date, create_uid, write_uid
        )
        SELECT partner_name, email_from, email_cc, partner_phone,
               linkedin_profile, type_id, priority,
               availability, create_date, write_date, create_uid, write_uid
        FROM hr_applicant
        WHERE candidate_id IS NULL;
```